platform/pulpcore/app/views/content.py

```diff
@@ -0,0 +1,151 @@
+import os
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.servers.basehttp import FileWrapper
```

Comment:
I couldn't create migrations.

`ImportError: cannot import name 'FileWrapper'`

FileWrapper is not available in Django 1.9 according to [this on StackOverflow](https://stackoverflow.com/questions/35817597/django-core-servers-basehttp-filewrapper-disappears-in-django-1-9)